lms/djangoapps/course_home_api/progress/v1/tests/test_views.py

```diff
@@ -97,3 +102,14 @@ def test_masquerade(self):
         # Masquerade as verified user
         self.update_masquerade(username=verified_user.username)
         assert self.client.get(self.url).data.get('enrollment_mode') == 'verified'
+
+    @patch.dict('django.conf.settings.FEATURES', {'DISABLE_START_DATES': False})
```

Comment:
Interesting, what is this doing?

Reply:
it is necessary for the tests using the start date transformer including this one to work properly, it is used in the start date transformer test

Reply:
Wild. Maybe just a misleading feature name. Or confusing why disable_start_dates would default to true. Seems like a desired feature.